libcontainer/cgroups/fs/cpu.go

```diff
@@ -84,6 +84,28 @@ func (s *CpuGroup) Set(path string, r *configs.Resources) error {
 			period = ""
 		}
 	}
+
+	var burst string
+	if r.CpuBurst != 0 {
```

Comment:
If we want to clear `cpu-burst`, the value should be 0. So we should change `r.CpuBurst` to a pointer and to check whether this is nil or not.

Reply:
I think splitting the nil check and zero-value check process is unnecessary here.

In my opinion, the nil value(the user doesn't input the burst value) and the zero-value(the user input burst as zero) means the same thing for us which is we should not write the burst value to the cgroup file

Reply:
For example, the container `test` started with the config:
```
cat config.json | jq .linux.resources.cpu
{
  "burst": 1000
}
```

After some time, we want to clear burst value, we may run:
```
runc update --cpu-burst 0 test
```